helios.js

```diff
@@ -24917,7 +24926,7 @@ export class Tx extends CborData {
 			this.#body.toCbor(),
 			this.#witnesses.toCbor(),
 			CborData.encodeBool(this.#valid),
-			CborData.encodeNull(),
+			this.#metadata === null ? CborData.encodeNull() : this.#metadata.toCbor(),
```

Comment:
the `fromCbor` counterpart is missing, are you planning to add that as part of this PR?